### ISSUE TITLE
Update usage_examples.rst and paper_examples.rst

### ISSUE THREAD

**Issue by ahy3nz** [mosdef-hub/foyer#265]
**Bug summary**
Update syntax for new mbuild/foyer organization

**Comment by umesh-timalsina** [comment]
Is the file `oplsaa_alkane.xml` accessible by `get_fn`?

**Comment by daico007** [comment]
I just tried it and the `get_fn` could not get the `oplsaa_alkane.xml`. Should we have a copy of `oplsaa_alkane.xml` on the `test/files/` directory?

**Comment by umesh-timalsina** [comment]
Yeah. The files are in `docs/examples/*`.

**Comment by daico007** [comment]
I think we can either move every file of that type to `test/files` or at least have a copy of them over there. What do you guys think?

**Comment by umesh-timalsina** [comment]
Also, while running Example2, I am getting the following error:
```
foyer.exceptions.MultipleValidationError: 

	Atom type opls_136 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
	Atom type opls_961 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
	Atom type opls_136 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
	Atom type opls_965 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
	Atom type opls_136 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
	Atom type opls_961 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
	Atom type opls_140 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
	Atom type opls_965 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
	Atom type opls_136 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
	Atom type opls_136 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
	Atom type opls_136 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
	Atom type opls_961 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
	Atom type opls_136 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
	Atom type opls_136 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
	Atom type opls_140 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
	Atom type opls_961 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
```

**Comment by ahy3nz** [comment]
> Also, while running Paper Example2, I am getting the following error:
> 
> ```
> foyer.exceptions.MultipleValidationError: 
> 
> 	Atom type opls_136 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
> 	Atom type opls_961 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
> 	Atom type opls_136 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
> 	Atom type opls_965 is found in RBTorsionForce at line 1262 but undefined in AtomTypes
> 	Atom type opls_136 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
> 	Atom type opls_961 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
> 	Atom type opls_140 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
> 	Atom type opls_965 is found in RBTorsionForce at line 1263 but undefined in AtomTypes
> 	Atom type opls_136 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
> 	Atom type opls_136 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
> 	Atom type opls_136 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
> 	Atom type opls_961 is found in RBTorsionForce at line 1264 but undefined in AtomTypes
> 	Atom type opls_136 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
> 	Atom type opls_136 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
> 	Atom type opls_140 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
> 	Atom type opls_961 is found in RBTorsionForce at line 1265 but undefined in AtomTypes
> ```

This issue arises because the `RBTorsionForce` section references certain atomtypes like `opls_136` that are not defined earlier in the `AtomTypes` section. 

I believe the fix here should be to prune the `RBTorsionForce` section to eliminate any lines that contain these unused, unspecified atomtypes

**Comment by ahy3nz** [comment]
> I think we can either move every file of that type to `test/files` or at least have a copy of them over there. What do you guys think?

Okay so for this, we could use the existing `get_fn` which, as you mention, looks in the `test/files` directory. Maybe moving the xmls there is easiest?

If we want to keep the `files` dir for the examples separate from the actual testing directory, there could be another utility function that looks in another directory for those files?

**Comment by uppittu11** [comment]
Fixed in PR #266
